wolox-books/src/app/screens/sign-up/sign-up.component.spec.ts

```diff
@@ -1,25 +1,202 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { MockUserService } from './mocks/mock-user-service';
+import { User } from './interfaces/user';
+import { MockUser } from './mocks/mock-user';
+import { RouterTestingModule } from '@angular/router/testing';
+import { UserService } from './../../services/user.service';
+import { SignBaseModule } from './../../components/sign-base/sign-base.module';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { SignUpComponent } from './sign-up.component';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
```

Comment:
Sort imports... package imports first, then local imports.
Example:
This is first `import { ComponentFixture, TestBed } from '@angular/core/testing';`
Then: `import { MockUser } from './mocks/mock-user';`